Tools/msgstringtable/msgstringtable_decoded.txt

```diff
@@ -3543,7 +3543,7 @@ MSI_ITEMSTORE_NOW_LOADING,"NOW LOADING..."
 MSI_RODEX_BUTTON_DELETE,"Delete"
 MSI_RODEX_BUTTON_ANSWER,"Reply"
 MSI_RODEX_BUTTON_SEND,"Send"
-MSI_RODEX_BUTTON_CONFIRM_ID,"Validate name"
+MSI_RODEX_BUTTON_CONFIRM_ID,"Check Name"
```

Comment:
I'd go with a simple, "Verify" or just keep it "Validate" since the error messages will tell you what's wrong anyways.